src/chgpasswd.c

```diff
@@ -424,6 +424,12 @@ int main (int argc, char **argv)
 	(void) bindtextdomain (PACKAGE, LOCALEDIR);
 	(void) textdomain (PACKAGE);
 
+#ifdef WITH_SELINUX
+	if (check_selinux_permit ("passwd") != 0) {
+		return (E_NOPERM);
+	}
+#endif
```

Comment:
Considering the overall coding style, should there be closing comments like `/* WITH_SELINUX */`?

Reply:
Sure, I can add those, some ifdefs had it and some not.